test/RenPool.test.js

```diff
@@ -36,29 +36,30 @@ describe('RenPool contract test', function () {
   const darknodeRegistry = new Contract(darknodeRegistryAddr, DarknodeRegistryLogicV1.abi);
   const darknodePayment = new Contract(darknodePaymentAddr, DarknodePayment.abi);
 
-  let owner, nodeOperator, alice, bob;
+  let _, owner, nodeOperator, alice, bob;
   let renPool;
 
   let snapshotID;
 
   before(async () => {
-    [owner, nodeOperator, alice, bob] = await getSigners();
+    [_, owner, nodeOperator, alice, bob] = await getSigners();
```

Comment:
Just for my understanding,
in the commit message https://github.com/Ethernautas/renpool/pull/63/commits/d12edc64f112243dabca9a0ad0006ed008b805c4,
when you mention _lodash signer_ , I'm guessing you're referring to the `_` symbol, but this is not related to _lodash_ the library. Is this correct?

Reply:
that's correct, it's not related to the lodash library, just borrowed the name bc it's short but we can change it if you think it's a bit confusing

Reply:
Oh no, I'm fine with the `_`. I was just confused by the `loadash` thing.